docker-local/docker/wait4/wait4.config.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
My only thought is this could be a `.js` file instead with a `module.exports` at the top, that would mean we can add comments in the file

Reply:
yes, you are right .js config will be much more flexible! fixing...